tests/test_database.nim

```diff
@@ -47,66 +53,96 @@ suite "Database Test Suite":
         check:
             db.has(key)
 
-    test "recover works":
-        let vals = @[@[byte 1, 2, 3, 4], @[byte 1, 2, 3, 4, 5], @[byte 1, 2, 3, 4, 5, 6]]
+proc testRecoverWorks(fn: InitProc) =
+    var db = fn()
+    
+    let vals = @[@[byte 1, 2, 3, 4], @[byte 1, 2, 3, 4, 5], @[byte 1, 2, 3, 4, 5, 6]]
```

Comment:
If this is just statically specified, why the nested-`seq`/`@`-construction rather than, e.g, just `[byte 1, 2, 3, 4, 1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 6]`?

And if one didn't want to statically specify it, there's still no need for the `@[foo]`.

Reply:
Because it is multiple values?

Reply:
Oops, indeed.